extension/chrome/settings/index.ts

```diff
@@ -309,6 +311,38 @@ Catch.try(async () => {
     }));
   };
 
+  const renderBlogPostList = async () => {
+    const errorRetrievingPosts = () => {
+      Xss.sanitizeAppend('.blog_post_list', `<div class="line>Error retrieving blog posts...try again later.</div>\n`);
+    };
+    try {
+      const res = await Api.ajax({
+        url: 'https://flowcrypt.com/feed',
+        method: 'GET',
+        dataType: 'json',
+        async: true
+      }, Catch.stackTrace());
+
+      interface BlogPostData {
+        url: string;
+        title: string;
+        date: string;
+      }
+      if (res instanceof Array) {
+        (res as BlogPostData[]).map(post => {
```

Comment:
Don't cast, the return method in the Api class would already have a return type, or you would cast it there.

Reply:
Also, please don't use map for looping when you can do
```js
for (const post of res) {
  // ...
}
```